runtime/src/serde_snapshot/future.rs

```diff
@@ -104,7 +105,7 @@ impl From<DeserializableVersionedBank> for BankFieldsToDeserialize {
             ns_per_slot: dvb.ns_per_slot,
             genesis_creation_time: dvb.genesis_creation_time,
             slots_per_year: dvb.slots_per_year,
-            unused: dvb.unused,
+            consumed_compute_units: dvb.consumed_compute_units,
```

Comment:
Sneaky! I was wondering how this was going to be handled without breaking snapshot compatibility.